sda-commons-server-morphia/README.md

```diff
@@ -97,10 +97,11 @@ mongo:
   username: "${MONGODB_USERNAME:-}"
   password: "${MONGODB_PASSWORD:-}"
   useSsl: ${MONGODB_USE_SSL:-true}
-  caCertificate: "${MONGODB_CA_CERTIFICATE:-}"
```

Comment:
As far as I remember, the option to pass the certificate via env string should still be possible, right?

Reply:
We deprecated it in the code ([MongoConfiguration](https://github.com/SDA-SE/sda-dropwizard-commons/commit/67d1c87b7a1c6425bbe0af42f6972351b0d2f923#diff-b8c0229f8e7925f373c12ffc9bd2322b763ddd8bbc0a1db9457a69dd28d52e90R63)), that's why thought to remove from the example config. Not sure if we should delete it or add the note for deprecation in the README?

Reply:
Yes, we should still support and mention the existing approach.

Reply:
Do we really require a deprecation here? I mean for simple deployments it should be still a possibility to pass the certificate via string.

Reply:
then do you think that we should give this in a second exemple and just propose it as an alternative ?

Reply:
As the environment variable / config.yaml option it is still available, we should keep the documentation of it.
I think we should deprecate it, so that it can be removed with the next major release of sda-dropwizard-commons.
If nothing special (namely providing a configuration option) is done by the implementor of a service, the MorphiaBundle will catch up certificates from the default directory of the CaCertificatesBundle. That is a different approach than the configuration. That's because I would mention it in a separate sentence (independent, of the config.yaml example) and document the default directory as mentioned in [the other comment](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1065#discussion_r673091124).

Reply:
Passing the certificate is more error-prone but giving the user of that bundle a less complex possiblity to choose how the certificate is passed. Are there any reasons why we should omit the option? At the end we'll force our customers to change something in there deployment, even if they are happy with that option right now.

Reply:
I'll ask the other maintainers for their preference regarding deprecation.